### DISCUSSION TITLE
Is it possible to Convert red6 jar file to .net dll file?

### DISCUSSION THREAD

**Discussion by uday0804** [red6/pdfcompare#130] [General]
Hello all,

Can someone provide an answer for the above question? Is it possible to convert red6/pdf compare jar file into .net dll. my company is currently want to user this tool and wanted to convert this into dll. I need to provide an answer whether it is possible or not. Help is really appreciated.

**Comment by finsterwalder** [comment]
PdfCompare is written in Java, that's why it's a jar file. You would have to wrap the jar file into a DLL somehow. There might be tools for that. You would have to research that yourself, since that is a general question, unrelated of PdfCompare.
